alpaca_trade_api/stream.py

```diff
@@ -206,19 +207,17 @@ async def _run_forever(self):
                 if retries > int(os.environ.get('APCA_RETRY_MAX', 3)):
                     await self.close()
                     self._running = False
-                    raise ConnectionError("max retries exceeded")
+                    log.warn('data websocket error, restarting connection: ' +
```

Comment:
I think APCA_RETRY_MAX and APCA_RETRY_WAIT are outdated now, since we actually retry anyway.